be/src/exec/spill/input_stream.cpp

```diff
@@ -120,6 +121,7 @@ class RawChunkInputStream final : public SpillInputStream {
 private:
     size_t read_idx{};
     std::vector<ChunkPtr> _chunks;
+    Spiller* _spiller = nullptr;
```

Comment:
it seems no need to define a Spiller field, instead, we just need field to save the counter or the metrics from Spiller